cmd/clusterctl/client/cluster/objectgraph_test.go

```diff
@@ -984,6 +989,39 @@ var objectGraphsTests = []struct {
 			},
 		},
 	},
+
+	{
+		name: "Cluster and External Object",
```

Comment:
I think we should have a test for Global & Namespaced external objects also here

Reply:
fixed